backend/fms_core/tests/invalid_templates/rename_invalid.csv

```diff
@@ -5,4 +5,4 @@ Naming Rules,,,,
 ,,,,
 ,,,,
 #,Old Container Barcode,New Container Barcode,New Container Name,Update Comment
-1,box001,box001$,original_box_2,dollars
\ No newline at end of file
+1,box001,rack001,original_box_2,dollars
```

Comment:
i would change the test from testing $ in barcode, to testing () in name.

Reply:
box001() ? it tests for invalid names (rack001 already exists and is not the same type of container) box001() would be an acceptable name, or () are still not permitted ?

Reply:
yeah i think you are right that this is a good test ... but maybe you can test the name validator ... () in the name is still not allowed as far as i know...

Reply:
() is allowed..

Reply:
it should not : name_validator = RegexValidator(re.compile(r"^[a-zA-Z0-9.\-_]{1,200}$"))

Reply:
we should open a different ticket regarding this. I think my PR still holds for what it is, and we could then later on add test with  () in container barcodes

Reply:
yeah of course ... this was just to fix the issues with the change in rename... i was merely proposing adding the test to name but if we are to change that too ... might as well wait for a final decision on the subject.